internal/ingress/controller/template/template.go

```diff
@@ -196,6 +196,8 @@ func buildLuaSharedDictionaries(s interface{}, dynamicConfigurationEnabled bool,
 			"lua_shared_dict balancer_ewma 1M",
 			"lua_shared_dict balancer_ewma_last_touched_at 1M",
 			"lua_shared_dict sticky_sessions 1M",
+			"lua_shared_dict cert_data 10M",
+			"lua_shared_dict key_data 10M",
```

Comment:
why different dictionaries?

Reply:
Why not use `configuration_data` for keeping certificates? My initial plan was to keep every config piece in there. Any reason to not use single dictionary for all?

Reply:
Sure, I just thought it was cleaner to have different dictionaries, but I can update it to just use the configuration_data dictionary